keps/sig-cluster-lifecycle/0035-20190128-addons-via-operators.md

```diff
@@ -0,0 +1,222 @@
+---
+kep-number: 35
+title: Addons via Operators
+authors:
+  - "@justinsb"
+owning-sig: sig-cluster-lifecycle
+reviewers:
+  - @luxas
+  - @roberthbailey
```

Comment:
Is @roberthbailey still going to be a reviewer for this considering his role change?